### PR TITLE
Add jax.numpy.square frontend function

### LINKED ISSUE #7775: jax.numpy.square

### PR DESCRIPTION
**Pull request by div5252**
Close #7775

### CODE REVIEW

**Comment by div5252** [comment]
@RickSanchezStoic you can have a look at this PR

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by div5252** [comment]
Ivy maintainers can review this

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by hirwa-nshuti** [comment]
Hi there 🙌

This function was completed via [commit](https://github.com/unifyai/ivy/commit/f4f3f8ebdbede01ef54d10c98cfe909c7afbd6e0).

This slipped under our radar and we kept the function open for contribution.
We are closing this PR as a duplicate and we are sorry for the inconvenience caused🙂

Feel free to take on another task for contribution.
Thanks
